apps/hash-ai-worker-py/app/__main__.py

```diff
@@ -0,0 +1,64 @@
+import asyncio
+import os
+from temporalio.worker import Worker
+from temporalio.client import Client
+from aiohttp import web
+
+from app.workflows import DemoWorkflow
+from app.activities import complete
+
+from dotenv import find_dotenv, load_dotenv
+
+
+load_dotenv()
+load_dotenv(dotenv_path=find_dotenv(filename=".env.local"), override=True)
```

Comment:
I think this `override` is what I've seen causing me problems in our code elsewhere, I think it actually overrides your environment using the file. Whereas I think the hierarchy we want is `.env` < `.env.local` < environment.

If we remove `override` does it just work?

Reply:
It does seem to work fine without the `override`. It correctly loads the `.env` and the `.env.local` while preserving the environment passed in explicitly.
The docs specify that the env will be overridden https://saurabh-kumar.com/python-dotenv/reference/

Reply:
From [the docs](https://pypi.org/project/python-dotenv/):

> With `load_dotenv(override=True)` or `dotenv_values()`, the value of a variable is the first of the values defined in the  > following list:
> - Value of that variable in the .env file.
> - Value of that variable in the environment.
> - Default value, if provided.
> - Empty string.
> With `load_dotenv(override=False)`, the value of a variable is the first of the values defined in the following list:
> - Value of that variable in the environment.
> - Value of that variable in the .env file.
> - Default value, if provided.
> - Empty string.

I think we should remove the override
```suggestion
load_dotenv(dotenv_path=find_dotenv(filename=".env.local"))
```